components/apps/ddm/DDM.tsx

```diff
@@ -45,7 +45,7 @@ export default function DDM(props) {
     setCookie('ddm-address', ddmAddress);
   };
 
-  const [getReplicationsConfig, setGetReplicationsConfig] = React.useState({ limit: 100 } as GetReplicationsConfig);
+  const [getReplicationsConfig, setGetReplicationsConfig] = React.useState({ limit: parseInt(localStorage.getItem("settings.replications.limit")) || 100 } as GetReplicationsConfig);
```

Comment:
nit: I think we should default it to `10` as the smallest option so it's less overwhelming and the user can scale up as they need - that's how Boost does it

Reply:
default should now be 10 👍